src/css/fonts.scss

```diff
@@ -17,3 +17,45 @@
   font-style: normal;
   font-weight: 700;
 }
+
+@font-face {
```

Comment:
![image](https://user-images.githubusercontent.com/13414205/83019992-847ed380-a030-11ea-894f-d68ce7d99730.png)
Do not applied correctly to the all elements  in the application. Check it with any font inspector

Reply:
As I know I should change only text fonts here

@kazurus

Reply:
As I remember, we have another font for code. But all other text should be with new font. @dudiq could you remind me about code font?

Reply:
@kazurus  `JetBrains Mono`

Reply:
and `JetBrains Mono` font should used for `<code>` `<pre>` and `.code`

Reply:
code font should stay unchanged